dist/templates/ovnkube-alerts.yaml.j2

```diff
@@ -347,5 +347,15 @@ spec:
          {{ range query "ovn_controller_integration_bridge_geneve_ports_total{job='ovnkube-node', namespace='ovn-kubernetes'} !=  scalar(count(kube_node_info)) - 1" }}
          {{ .Labels.instance }}: {{ .Value }}
          {{ end }}.
- 
+
+    - alert: OVNKubeAllocatedSubnetsDoNotMatch
+      expr: |
+        ovnkube_clustermanager_allocated_v4_host_subnets{job="ovnkube-master", namespace="ovn-kubernetes"} !=
+        scalar(sum(kube_node_info))
+      for: 10m
+      labels:
+       severity: major
+      annotations:
+       message: |
+         ovnkube clustermanager allocated IPv4 host subnets value for {{ $labels.network_name }} network should be same as the number of K8s nodes
```

Comment:
what about IPv6?

Reply:
yeah, gonna add the same for ipv6 too. Thanks